SocketRocket/SRWebSocket.m

```diff
@@ -255,6 +255,7 @@ @implementation SRWebSocket {
     NSString *_closeReason;
     
     NSString *_secKey;
+    NSString *_authorization;
```

Comment:
If you're feeling super helpful, a better name would be something like `_basicAuthorizationString`.